js/main.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable max-classes-per-file */
+
+class Collection {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const today = new Date().toUTCString();
+const time = document.querySelector('.time');
+time.textContent = today;
+
+const bookTitle = document.querySelector('.title');
+const bookAuthor = document.querySelector('.author');
+const addBtn = document.querySelector('.btn');
+const navList = document.querySelector('.list');
+const bookList = document.querySelector('.bookListSection');
+const navAddBook = document.querySelector('.addBook');
+const bookCreation = document.querySelector('.bookCreationSection');
+const navContact = document.querySelector('.contact');
+const contactInfo = document.querySelector('.contactSectionNav');
+
+const data = JSON.parse(localStorage.getItem('list'));
+
+class Library {
+  static myBookList = [];
+
+  static showList() {
+    bookList.style.display = ('block');
+    bookCreation.style.display = ('none');
+    contactInfo.style.display = ('none');
+  }
+
+  static createBooks() {
+    bookCreation.style.display = ('flex');
+    bookList.style.display = ('none');
+    contactInfo.style.display = ('none');
+  }
+
+  static showContactSection() {
+    contactInfo.style.display = ('flex');
+    bookList.style.display = ('none');
+    bookCreation.style.display = ('none');
+  }
+
+  static addNewBook() {
```

Comment:
[OPTIONAL] I think you could improve the user experience of adding a new book to the library by moving the user to the booklist once the book has been created or displaying a hint that book has been successfully created.

Also, using the event `submit` instead of `click` on the add button would allow the user to add books using the key `enter` instead of having to manually click the button every time.